From df5d43af207c4056daf95f125f62d94e0509709e Mon Sep 17 00:00:00 2001
From: Markus Volk <f_l_k@t-online.de>
Date: Sun, 10 Apr 2022 06:41:21 +0200
Subject: [PATCH] puae-libretro: fix math.h include

/home/flk/build/poky/build-intel/tmp/work/corei7-64-poky-linux/puae-libretro/2021+gitAUTOINC+8c44289acf-r0/recipe-sysroot/usr/include/bits/mathcalls.h:133:1: error: expected ';' before 'extern'
|   133 | __MATHCALL_VEC (log2,, (_Mdouble_ __x));

math.h seems to get included multiple times
---
 libretro/libretro-core.h | 1 -
 sources/src/audio.c      | 2 --
 sources/src/custom.c     | 1 -
 sources/src/gfxutil.c    | 2 --
 4 files changed, 6 deletions(-)

diff --git a/libretro/libretro-core.h b/libretro/libretro-core.h
index d5dcdbe..7ea0ec7 100644
--- a/libretro/libretro-core.h
+++ b/libretro/libretro-core.h
@@ -3,7 +3,6 @@
 
 #include <stdint.h>
 #include <string.h>
-#include <math.h>
 #include <stdlib.h>
 #include <stdio.h>
 #include <stdbool.h>
diff --git a/sources/src/audio.c b/sources/src/audio.c
index 2fcd200..bd8a339 100644
--- a/sources/src/audio.c
+++ b/sources/src/audio.c
@@ -43,8 +43,6 @@
 #endif
 #include "threaddep/thread.h"
 
-#include <math.h>
-
 #define DEBUG_AUDIO 0
 #define DEBUG_AUDIO_HACK 0
 #define DEBUG_CHANNEL_MASK 15
diff --git a/sources/src/custom.c b/sources/src/custom.c
index de25da3..4a9d04f 100644
--- a/sources/src/custom.c
+++ b/sources/src/custom.c
@@ -13,7 +13,6 @@
 
 #include <ctype.h>
 #include <assert.h>
-#include <math.h>
 
 #include "options.h"
 #include "uae.h"
diff --git a/sources/src/gfxutil.c b/sources/src/gfxutil.c
index 0ab3ff0..a489a68 100644
--- a/sources/src/gfxutil.c
+++ b/sources/src/gfxutil.c
@@ -14,8 +14,6 @@
 #include "xwin.h"
 #include "gfxfilter.h"
 
-#include <math.h>
-
 float getvsyncrate(int monid, float hz, int *mult)
 {
 	struct amigadisplay *ad = &adisplays[monid];
-- 
2.25.1

